network/src/proto/consensus.proto

```diff
@@ -117,6 +125,14 @@ message Vote {
   types.LedgerInfo ledger_info = 6;
   // Signature of the ledger info.
   bytes signature = 7;
+  // The id of the parent block of the proposed block
```

Comment:
Dumb question since it's been a while since I've looked at this codebase. Are we worried at all about the duplication between the Vote and QC protos? Also given the importance of canonical serialization, is it worth considering the same thing we've looked at doing for transactions where we make the canonical serialization format the one we use to pass the structure over the wire. (i.e. if we sign votes using a canonical serialization that appends the version/round/parent/etc simply serializing those bytes over the wire to represent the vote)

Reply:
Good point! We should reorganize the Vote and QC structures: in the end a QC is merely a vote with more signatures :). It's probably orthogonal to the purpose of this PR though. Opened issue #402 to fix it in a separate change.